app/src/main/java/org/cru/godtools/fragment/ToolsFragment.java

```diff
@@ -235,7 +238,12 @@ private void updateVisibleBanner() {
                     LocaleUtils.getDeviceLocale(requireContext()).getLanguage().equals(Locale.ENGLISH.getLanguage())) {
                 mToolsHeaderAdapter.setBanner(BannerType.TUTORIAL_TRAINING);
                 mToolsHeaderAdapter.setPrimaryCallback(b -> openTrainingTutorial());
-                mToolsHeaderAdapter.setSecondaryCallback(b -> settings.setFeatureDiscovered(FEATURE_TUTORIAL_TRAINING));
+                mToolsHeaderAdapter.setSecondaryCallback(b -> {
+                    EventBus.getDefault().post(new TutorialAnalyticsActionEvent(
+                            TutorialAnalyticsActionEventKt.ADOBE_TUTORIAL_HOME_DISMISS)
+                    );
```

Comment:
just double checking, you want the action name to be `cru.tutorial_home_dismiss`?

Reply:
https://docs.google.com/spreadsheets/d/1YLkGQ8ptXWn6xmB82kzUYIkXHsI-BsnA2iQanQdPcnw/edit#gid=1544271866&range=244:244

This is the link to the document reference.